packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx

```diff
@@ -32,11 +34,19 @@ export default {
     },
   },
   argTypes: {
+    action: {
+      control: "text",
+      default: null,
+    },
```

Comment:
Could this be renamed to `actionText` or `callToActionText`? I thought it was a function until I scrolled up and looked at the arg type.

Reply:
So, the control here is `text`, but the anticipated value is actually a full React block like `<Link href="https://whatever.com">Whatever</Link>`. You can see the example in the dismissible section.

When we wrap, we could separate this into `actionText` and `actionLink`? I didn't want to add that complexity without talking it through with you, though.

Reply:
Yeah, we can separate it, but it should probably be `actionUrl` or something for the link since `actionLink` could still refer to a component.